blt/src/Blt/Plugin/Commands/MultisiteCommands.php

```diff
@@ -14,8 +14,6 @@
 use AcquiaCloudApi\Endpoints\SslCertificates;
 use Consolidation\AnnotatedCommand\CommandData;
 use Consolidation\AnnotatedCommand\CommandError;
-use GuzzleHttp\Client as GuzzleClient;
-use GuzzleHttp\Exception\RequestException;
```

Comment:
Just checking that this change was intentional?

Reply:
Yes - I commonly run `./vendor/bin/phpcbf` which tries to automatically fix PHPCS violations. Then I run `./vendor/bin/phpcs` after that to fix what it couldn't. I guess these classes were not in use there.